### ISSUE TITLE
Failure to Compile - Missing Module/Corrupted Files?

### ISSUE THREAD

**Issue by Dumpshock** [zephyris/opengfx2#1]
Am attempting to compile this, but running into some issues.

Do you have additional modules/dependencies you may have forgotten to mention? Found that you were also doing an import on '**blend_modes**' that needed to be deployed to clear some errors, this was not mentioned in your description.

Current hurdle is that PIL seems to not be able to identify the image files with the following error targeted at the .PNG files: "PIL.UnidentifiedImageError: cannot identify image file"

Tried opening them up on using Paint.Net and Gimp and none of them will open. Best answer I've been able to find is something malformed in the images themselves, perhaps corrupted on upload to the Git?

On the side, any chance of a compile from your side with an upload into releases for those that aren't able to get this compiled?

**Comment by Dumpshock** [comment]
After some sleep and taking another look at this, I found that the issue seems to have been me not understanding Git LFS and attempting to pull the archive by Zip.

However, it seems that I am still unable to pull the LFS data in order to compile  as the repo is over it's data quota.

"This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access."

Seen some menion that enabling "Include Git LFS objects in archives " in the repo's  Settings > Archive as a solution to LFS's design. Barring that, any chance of compiling a release to test, or any alternatives you're aware of?

**Comment by zephyris** [comment]
Hey, sorry about that. I think you're right - looks like it's down to the LFS limits. If you've got any specific knowledge/suggestions for LFS settings for `.gitattributes` or anywhere else I'm very happy to try them... I'm afraid I'm "learning by doing"!

For now, I've posted the compiled grfs on the tt-forums: https://www.tt-forums.net/viewtopic.php?p=1259747&sid=43d8e988643c1dfbb6698a4662b24025#p1259747

**Comment by zephyris** [comment]
I've dug into the `Settings > Archive` suggestion, and can't find an Archive section anywhere! This unfortunately wouldn't be an immediate solution, archive download counts against LFS bandwidth allowance too. The immediate solution is to give Github some $$$...

**Comment by Dumpshock** [comment]
Hey, thanks for uploading the compiled .grfs, can't wait to take a look. I pulled some of your older ones and am really liking what I've been seeing. Having tried a few of the other 32bpp sets, they really don't compare to the OpenGFX ones IMO, but having that extra detail is really nice.  Any chance the same treatment is planned for signals? They can be a bit hard to read, getting extra zoom in those would be neat.

As for Git LFS, I don't have any particular suggestions as I only learned it existed through your repo. I'd just read from other posts talking about similar issues that it was suggested to change that setting. How large are they? Could they be mirrored to a online file storage such as Google Drive?
